_rules/lang-valid-de46e4.md

```diff
@@ -23,7 +23,7 @@ This rules applies to each HTML or SVG element that is a [descendant](https://do
 
 ## Expectation
 
-The `lang` and `xml:lang` attributes of the test target have a [valid language subtag](#valid-language-subtag) if the attribute is not empty ("").
+For each test target, the `lang` and `xml:lang` attributes have a [valid language subtag](#valid-language-subtag) if the attribute is not empty ("").
```

Comment:
`lang` and `xml:lang` > `lang` or `xml:lang`
(to match the applicability)

Reply:
Interesting point. I think it can go either way. That's not related to this change though. Please open a separate issue.